molgenis-core-ui/src/main/java/org/molgenis/core/ui/MolgenisMenuController.java

```diff
@@ -31,6 +32,10 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.servlet.HandlerMapping;
 
+/**
+ * Looks up the plugin that will handle the request. Adds context attributes to the Model so the
```

Comment:
Is this comment a comment on the class or a comment beloning to a method of this class ?

Reply:
I've added more javadoc to the rest of the class with what I understand of it.